halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/kubernetes/v2/KubernetesV2Service.java

```diff
@@ -89,7 +89,7 @@ default boolean isEnabled(DeploymentConfiguration deploymentConfiguration) {
   }
 
   default List<String> getReadinessExecCommand(ServiceSettings settings) {
-    return Arrays.asList("wget", "--no-check-certificate", "--spider", "-q", settings.getScheme() + "://localhost:" + settings.getPort() + settings.getHealthEndpoint());
+    return Arrays.asList("wget", "--no-check-certificate", "--spider", "-q", settings.getScheme() + "://localhost:" + settings.getPort() + settings.getHealthEndpoint(), settings.getKubernetes().getReadinessCommandExtras());
```

Comment:
Thoughts on making this fully overridable? @jgramoll @lwander

Reply:
I like that idea

Reply:
Looks like someone else was looking into this too

https://github.com/spinnaker/halyard/pull/1171/files